components/Datepicker/src/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import clsx from 'clsx';
+import {
+  format,
+  Locale,
+  getWeeksInMonth,
+  formatISO,
+  setDay,
+  addWeeks,
+  subMonths,
+  addMonths,
+  isSameMonth,
+  setDate,
+} from 'date-fns';
+import { nl } from './locale';
+
+import BaseProps from '@gemeente-denhaag/baseprops';
+
+import './datepicker.css';
+
+export interface DatepickerProps extends BaseProps {
```

Comment:
I wonder if we want to use ` extends InputHTMLAttributes<HTMLInputElement>` instead. This will make sure we can support all the expected attributes on an input, and pass them to the native input within our wrapper.

Reply:
Changed :)